apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -14,12 +14,18 @@ export default class Comment extends Component {
   };
 
   state = {
-    isShowingCommentOptions: false
+    isShowingCommentOptions: false,
+    hasError: false
```

Comment:
Do we need `hasError` in state? From what I can tell the `hasError` property is supplied through props via the comment object only right?

Reply:
Yep I think that is some cleanup I missed, thanks!